target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -68,60 +74,67 @@ endef
 TARGET_DEVICES += linksys_wrt1900ac
 
 define Device/plathome_openblocks-ax3-4
+  DEVICE_VENDOR := Plat'Home
+  DEVICE_MODEL := OpenBlocks AX3
```

Comment:
Would be interesting to know why there is a four appended everywhere except for the title.

Reply:
4 stands for four gigabit ethernet ports. There are versions with less ports.

Reply:
Should we "add" a variant for that?
DEVICE_VARIANT := 4 ports

Reply:
Sounds good. The (auto-translated) product site speaks about the port 2 and port 4 versions.